### DISCUSSION TITLE
Why the Peak Time is diferent from real time?

### DISCUSSION THREAD

**Discussion by leonneonn** [Chia-Network/chia-blockchain#1939] [Q&A]
Chia is running at my computer 24/7  but fore some hours pick time is not same as real time. Cpu works 60% and ram works around 50% so why this is happening?  For example:  now is 8pm peak time is 6:45 pm sometimes blocks even jumping to 4pm. Why this syncing is not ok special at evenings?

Update
after last plotting is finish suddenly time is ok. Before were jumping from 8pl to  4pm to 6pm to 7 pm to 5 pm. Procesor and ram were not full I was just plotting two plots.

**Comment by L3Sota** [comment]
Were you plotting with your OS disk as temp space? Maybe Chia was in a race with the plotter to get disk access.

**Reply by leonneonn** [reply]
Im plotiong on ssd nvme 1T. Only sometimes I have problem with delay and jumping on different peak time dont know why. Im ploting 2 plots. After restart to morning was ok again . Im ploting same all the time and sometimes this is happening and I cant find logiczne reason for it.  I dont want to check  plotting  all the time if this is happening. Im just loosing chance to win chia
